_data/compatibility/conio.yaml

```diff
@@ -0,0 +1,60 @@
+    platforms:
+      - iOS
+    version: "2.5.6"
+  features:
+    receive:
+      notification: "false"
+      list: "false"
+      details: "false"
+      shows_replaced_version: "true"
+      shows_original_version: "true"
+    send:
+      signals_bip125: "true"
+      list: "false"
+      details: "false"
+      shows_replaced_version: "untested"
+      shows_original_version: "untested"
+  examples:
+    - image: /img/compatibility/conio/rbf/send-screen-default.png
+      caption: >
+        Sending RBF Transaction - Default send transaction screen. No RBF options. Transaction is sent with RBF signaled.
+    - image: /img/compatibility/conio/rbf/transaction-details-sent.png
+      caption: >
+        Bumping RBF Transaction - Transaction details screen for a sent and unconfirmed transaction. Cannot get Send Faster button enabled even when funds to bump are available.
+    - image: /img/compatibility/conio/rbf/notification-incoming-rbf.png
+      caption: >
+        Bumping RBF Transaction - Notice of incoming transaction. No RBF flag.
+    - image: /img/compatibility/conio/rbf/transaction-list-incoming-rbf.png
+      caption: >
+        Bumping RBF Transaction - Transaction list screen showing incoming RBF enabled transaction. No RBF flag.
+    - image: /img/compatibility/conio/rbf/transaction-details-incoming-rbf.png
+      caption: >
+        Bumping RBF Transaction - Transaction details of incoming transaction. No RBF signaled. "Receive Faster" for CPFP.
+    - image: /img/compatibility/conio/rbf/transaction-details-incoming-rbf-2.png
+      caption: >
+        Bumping RBF Transaction - More transaction details.
+    - image: /img/compatibility/conio/rbf/transaction-details-incoming-rbf-3.png
+      caption: >
+        Bumping RBF Transaction - More transaction details. After a period of time the Receive Faster button was now enabled.
+    - image: /img/compatibility/conio/rbf/transaction-details-incoming-rbf-receive-faster.png
+      caption: >
+        Bumping RBF Transaction - Transaction details showing Receive Faster options.
+    - image: /img/compatibility/conio/rbf/notification-replacement.png
+      caption: >
+        Receiving Bumped RBF Transaction - Receiving bumped transaction shows “Incoming” amount as the sum of the original and replacement transaction.
+    - image: /img/compatibility/conio/rbf/transaction-list-incoming-replacement.png
+      caption: >
+        Receiving Bumped RBF Transaction - Both original and replacement transaction show in transaction list screen.
+    - image: /img/compatibility/conio/rbf/transaction-list-replacement-confirmed.png
+      caption: >
+        Receiving Bumped RBF Transaction - After replacement transaction confirms, the original is removed from the list.
+    - image: /img/compatibility/conio/rbf/transaction-details-replacement.png
+      caption: >
+        Receiving Bumped RBF Transaction - Additional transaction details available once transaction was confirmed.
```

Comment:
with no segwit section this comes up blank on the main page which is inconsistent with the other companies.


<img width="1123" alt="Screen Shot 2019-08-19 at 3 34 06 PM" src="https://user-images.githubusercontent.com/755825/63293824-f56daf80-c296-11e9-8864-656f82645e5d.png">

Reply:
@adamjonas I have not been able to test their wallet for the last couple months due to geographical restrictions. So we do not have data for them yet. 

Ive been working on gaining access and I did just get an iOS TestFlight invite. I will try to re-test to try and get their segwit tests in there.

Reply:
Although testing Coinio would be great, as a general solution we could probably just skip including tools without data in the listing, e.g.:

```diff
diff --git a/en/compatibility.md b/en/compatibility.md
index b4a239b..0467cf1 100644
--- a/en/compatibility.md
+++ b/en/compatibility.md
@@ -72,8 +72,6 @@ h1, h2, h3, h4, h5, h6 { text-align: center; }
       {% include functions/compat-cell.md state=tool.segwit.features.send.bech32 anchor="#segwit-send-bech32" %}
       {% include functions/compat-cell.md state=tool.segwit.features.send.bech32_p2wsh anchor="#segwit-send-bech32_p2wsh" %}
       {% include functions/compat-cell.md state=tool.segwit.features.send.change_bech32 anchor="#segwit-send-change_bech32" %}
-    {% else %}
-      <td colspan="7" class="default"></td>
     {% endif %}
   </tr>
 {% endfor %}
```

Reply:
@harding I had the idea that a consistent set of wallets/services would be a good idea. But I agree with you and @adamjonas that the empty row is probably worse than omitting missing tests. I opened https://github.com/bitcoinops/bitcoinops.github.io/pull/209 for this.

Reply:
Pushed a commit with Conio bech32 test results and screenshots and rebased.